src/makeRobot.js

```diff
@@ -38,7 +38,98 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    'name': name,
+    'wheels': wheels,
+    'version': version,
+    get info() {
+      return `name: ${this.name}, chip version: ${this.version},`
+      + ` wheels: ${this.wheels}`;
+    },
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    set location(loctn) {
+    },
+    goForward(parametr = 1) {
+      if (parametr < 0) {
```

Comment:
add space between object method 
```suggestion
    },
    coords: {
      x: 0,
      y: 0,
    },
    
    get location() {
      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
    },
    
    set location(loctn) {
    },
    
    goForward(parametr = 1) {
      if (parametr < 0) {
```